log-insights/limits.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'pganalyze Log Insights Limits'
+backlink_href: /docs/log-insights
+backlink_title: 'Log Insights'
+---
+
+Log Insights has a rate limit of 1 GB of log data per day for each server. Limits for
```

Comment:
Should we hint at increased limits being available by contacting us? (we have a rather basic way to do this today, but if there is a larger interest in it, we could make this a self-serve feature)

e.g. "Additional log limit quota can be added to your plan by contacting pganalyze Support.".

Reply:
Sure :+1: